modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -35,14 +35,14 @@ An administrator-defined group of server-nodes within the cluster is unresponsiv
 
 Auto-failover is triggered:
 
-* Only on the occurrence of one event at a time.
-If multiple events are concurrent, auto-failover is not triggered.
-* Sequentially, only up to an administrator-specified maximum number of events.
+* Only on the occurrence of one failure at a time.
+If multiple nodes fail concurrently, auto-failover is not triggered.
```

Comment:
The changes for 2225 haven't been merged in yet, so they're living on a separate branch.

Reply:
Since this is related to https://github.com/couchbase/docs-server/pull/2225, it's probably easier to review them together. I'll merge the changes from this branch into 2225